concourse/pipelines/guest-package-build.yaml

```diff
@@ -1596,6 +1596,489 @@ jobs:
       result_state: "failure"
       start_timestamp: ((.:start-timestamp-ms))
 
+- name: build-compute-image-windows
+  plan:
+  - get: compute-image-windows
+    trigger: true
+    params:
+      skip_download: true
+  - load_var: commit-sha
+    file: compute-image-windows/.git/ref
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: get-github-token
+    file: guest-test-infra/concourse/tasks/get-github-token.yaml
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: timestamp
+    file: timestamp/timestamp
+  - task: compute-image-windows-integration-test
+    file: guest-test-infra/concourse/tasks/package-gointegtest.yaml
+    vars:
+      repo-name: compute-image-windows
+      timestamp: ((.:timestamp))
+  - task: validate-integtest-result
+    file: guest-test-infra/concourse/tasks/validate-integtest-result.yaml
+    vars:
+      timestamp: ((.:timestamp))
+      repo-name: compute-image-windows
+  - task: compute-image-windows-gotest-coverage
+    file: guest-test-infra/concourse/tasks/gotest.yaml
+    input_mapping:
+      repo: compute-image-windows
+  - load_var: coverage-percent
+    file: gotest/coverage.txt
+  - task: generate-package-version
+    file: guest-test-infra/concourse/tasks/generate-package-version.yaml
+    input_mapping: {repo: compute-image-windows}
+  - load_var: package-version
+    file: package-version/version
+  - in_parallel:
```

Comment:
looks like 1 step, no need for `in_parallel`

Reply:
Fixed. Removed the over use of `in_parallel`